src/test/scala/is/hail/utils/RichIndexedRowMatrixSuite.scala

```diff
@@ -48,4 +49,11 @@ class RichIndexedRowMatrixSuite extends SparkSuite {
       idxRowMat.toBlockMatrix(2, 0)
     }
   }
+
+  @Test def testToLocalMatrix() {
+    val mat = new DenseMatrix(10, 10, (0.0 until 100.0 by 1.0).toArray)
+    val bm = from(sc, mat, 10, 10)
+    val localFromIRM = bm.toIndexedRowMatrix().toLocalMatrix()
+    TestUtils.assertMatrixEqualityDouble(localFromIRM.asBreeze(), mat.asBreeze())
```

Comment:
Unfortunately, you must have dense matrices to do this. We do, but I have to call `toDenseMatrix` on everything.